argus/util.py

```diff
@@ -24,11 +24,16 @@
 import subprocess
 import sys
 import time
+import unittest
 
 import six
 
+from argus import log as argus_log
+from argus.tests import base as base_testcase
```

Comment:
I would not add a unittest dependecy in the argus utils, as this module should be independent to other modules used in argus